docs/blog/08-21-2024-introducing-the-federation-api.md

```diff
@@ -0,0 +1,402 @@
+    if (files === null) {
+      continue;
+    }
+
+    for (let entry of files.values) {
+      entry.label = server;
+      result.push({
+        group: server,
+        value: entry.value,
+        count: entry.count
+      });
+    }
+  }
+
+  return result.filter((d) => d.count > 0);
+}
+
+const plotSubjectByCount = async (id: string, title: string, field: string, options: any) => {
+  let data = prepareByCount(await api.countSubjectsBy(field));
+  Graph.bar(id, title, data, options);
+}
+
+const plotSampleByCount = async (id: string, title: string, field: string, options: any) => {
+  let data = prepareByCount(await api.countSamplesBy(field));
+  Graph.bar(id, title, data, options);
+}
+
+const plotFileByCount = async (id: string, title: string, field: string, options: any) => {
+  let data = prepareByCount(await api.countFilesBy(field));
+  Graph.bar(id, title, data, options);
+}
+
+onMounted(async() => {
+  // api.samples((name, kind, values, complete) => {
+  //   if (values !== null) {
+  //     samples.value.push(...values);
+  //   }
+  // });
+
+  plotFileByCount(
+    "#fileByType",
+    "Files by Type",
+    "type",
+    {
+      scale: 1.05
+    }
+  );
+
+  plotSampleByCount(
+    "#sampleByStrategy",
+    "Samples by Library Strategy",
+    "library_strategy"
+  );
+
+  plotSubjectByCount(
+    "#subjectByRace",
+    "Subjects by Race",
+    "race",
+    {
+      margin: {
+        bottom: 150,
+      },
+    }
+  );
+});
+
+const osteosarcomas = computed(() => {
+  return samples.value.filter((s) => {
+    let diagnosis = s.metadata?.diagnosis?.value || null;
+
+    return diagnosis !== null &&
+           diagnosis !== undefined &&
+           diagnosis.toLowerCase().includes("osteosarcoma");
+  });
+});
+
+const neuroblastomas = computed(() => {
+  return samples.value.filter((s) => {
+    let diagnosis = s.metadata?.diagnosis?.value || null;
+
+    return diagnosis !== null &&
+           diagnosis !== undefined &&
+           diagnosis.toLowerCase().includes("neuroblastoma");
+  });
+})
+
+watch([osteosarcomas, neuroblastomas], ([osteosarcomas, neuroblastomas], _) => {
+  let osteoAges = osteosarcomas.map((d) => d.metadata?.age_at_diagnosis?.value || null);
+  let thresholds = d3.ticks(...d3.nice(...d3.extent(osteoAges), 10), 40)
+})
+</script>
+
+<ClientOnly>
+<div class="text-4xl font-extrabold">Introducing the CCDI Federation API</div>
+<div class="text-lg mt-1 text-slate-500">The CCDI Federation Developers</div>
+<div class="text-slate-400 italic">September 25, 2024</div>
+
+Today, we're announcing the launch of the Childhood Cancer Data Initiative
+(CCDI) Federation API 🎉! Foundationally, we've created this API to facilitate
+real-time discovery of research data being released across the pediatric cancer
+ecosystem. Our hope is that this API will enable pediatric cancer researchers
+around the world to quickly identify data that is relevant to their research and
```

Comment:
```suggestion
around the world to quickly identify data that are relevant to their research and
```

Reply:
There were a couple of suggestions regarding this; I went with what is suggested [here](https://www.thesaurus.com/e/grammar/data-is-or-data-are/) to make it singular:

> In general usage, data is treated as both a singular and a plural word. When data is used as a [mass noun](https://www.dictionary.com/browse/mass-noun?adobe_mc=MCMID%3D51888964765408706474184288583947612217%7CMCORGID%3DAA9D3B6A630E2C2A0A495C40%2540AdobeOrg%7CTS%3D1727205591) (a noun that can’t be counted, like sand) to mean “information,” it typically uses a singular verb.